.github/workflows/publish.yml

```
@@ -3,31 +3,45 @@ name: Publish
 # See also TRANSLATIONS.md.
 
 on:
+  pull_request:
   push:
     branches:
       - main
   workflow_dispatch:
 
-permissions:
-  contents: read
-  pages: write
-  id-token: write
-
-# Allow one concurrent deployment
-concurrency:
-  group: pages
-  cancel-in-progress: true
-
 env:
   CARGO_TERM_COLOR: always
   # Update the language picker in index.hbs to link new languages.
-  LANGUAGES: ar bn da de el es fa fr id it ja ko pl pt-BR ro ru tr uk vi zh-CN zh-TW
 
 jobs:
-  publish:
-    environment:
-      name: github-pages
-      url: ${{ steps.deployment.outputs.page_url }}
+  create-translation:
+    strategy:
+      matrix:
+        language:
+          [
+            "en",
+            "ar",
+            "bn",
```

From what I understand of YAML, this can be simplified to

```suggestion
        language:
          - "en"
          - "ar"
          - "bn"
```

and so on, which seems ever so slightly more high level.